cfuncs.py

```diff
@@ -8,12 +8,12 @@
 from ctypes.wintypes import HANDLE
 from ctypes.wintypes import LPVOID
 from ctypes.wintypes import LPCVOID
-from pywinauto.win32structures import SECURITY_ATTRIBUTES
-from pywinauto import win32defines
+from pywinauto.windows.win32structures import SECURITY_ATTRIBUTES
+from pywinauto.windows import win32defines
```

Comment:
Maybe duplicate these definitions from win32defines? There are not so many defines used.

Reply:
`SECURITY_ATTRIBUTES` structure from `win32structures.py` depends from the base classes from `base_types` module. 
Is it need to copy this structure and all its base classes to `cfuncs.py` too?

Reply:
I copied `SECURITY_ATTRIBUTES` also.